SharpDXtest/EditorWindow.xaml.cs

```diff
@@ -133,7 +133,7 @@ private void Window_Loaded(object sender, RoutedEventArgs e)
 
             copyFramebuffer = new FrameBuffer((int)ActualWidth, (int)ActualHeight);
 
-            EngineCore.CurrentScene = AssetsManager.LoadScene("Assets\\Scenes\\Scene5.xml");
+            EngineCore.CurrentScene = AssetsManager.LoadScene("Assets\\Scenes\\Bloom.xml");
```

Comment:
Do not change the starting scene in version for pull request, but instead bind opening of your scene to the button on GUI

Reply:
Done